fe/fe-core/src/main/java/org/apache/doris/statistics/StatisticsCollector.java

```diff
@@ -83,7 +74,14 @@ protected void createSystemAnalysisJob(AnalysisInfo jobInfo)
             analysisManager.createTableLevelTaskForExternalTable(jobInfo, analysisTasks, false);
         }
         Env.getCurrentEnv().getAnalysisManager().registerSysJob(jobInfo, analysisTasks);
-        analysisTasks.values().forEach(analysisTaskExecutor::submitTask);
+        for (BaseAnalysisTask task : analysisTasks.values()) {
+            try {
+                task.execute();
```

Comment:
Not a block. But it's better to write the successfully executed tasks' result to column statistics table, instead of fail the whole job when one task failed.

Reply:
Yes, I agree with you and the logic you mentioned has been implemented in `AnalysisJob`